docs/source/users/options/output_option.rst

```diff
@@ -20,4 +20,123 @@ Default is ``fitbenchmarking_results``
 .. code-block:: rst
 
     [OUTPUT]
```

Comment:
All the headers should be the same now -- either all `[OUTPUT]` or all `[PLOTTING]`